NorthstarDLL/server/r2server.h

```diff
@@ -9,102 +9,251 @@ namespace R2
+		char _unk34[92]; // 0x2284 ( Size: 92 )
+		bool m_wallHanging; // 0x22e0 ( Size: 1 )
+		char _unk35[11]; // 0x22e1 ( Size: 11 )
+		int32_t m_traversalType; // 0x22ec ( Size: 4 )
+		int32_t m_traversalState; // 0x22f0 ( Size: 4 )
+		char _unk36[40]; // 0x22f4 ( Size: 40 )
+		Vector3 m_traversalForwardDir; // 0x231c ( Size: 12 )
+		Vector3 m_traversalRefPos; // 0x2328 ( Size: 12 )
+		char _unk37[32]; // 0x2334 ( Size: 32 )
+		float m_traversalYawDelta; // 0x2354 ( Size: 4 )
+		int32_t m_traversalYawPoseParameter; // 0x2358 ( Size: 4 )
+		char _unk38[140]; // 0x235c ( Size: 140 )
+		bool m_grappleActive; // 0x23e8 ( Size: 1 )
+		char _unk39[19]; // 0x23e9 ( Size: 19 )
+		int32_t m_activeZipline; // 0x23fc ( Size: 4 )
+		bool m_ziplineReverse; // 0x2400 ( Size: 1 )
+		char _unk40[11]; // 0x2401 ( Size: 11 )
+		bool m_ziplineValid3pWeaponLayerAnim; // 0x240c ( Size: 1 )
+		char _unk41[3]; // 0x240d ( Size: 3 )
+		int32_t m_ziplineState; // 0x2410 ( Size: 4 )
+		char _unk42[392]; // 0x2414 ( Size: 392 )
+		float m_lastDodgeTime; // 0x259c ( Size: 4 )
+		char _unk43[8]; // 0x25a0 ( Size: 8 )
+		bool m_iSpawnParity; // 0x25a8 ( Size: 1 )
+		char _unk44[21]; // 0x25a9 ( Size: 21 )
+		bool m_isPerformingBoostAction; // 0x25be ( Size: 1 )
+		char _unk45[233]; // 0x25bf ( Size: 233 )
+		int32_t m_lastUCmdSimulationTicks; // 0x26a8 ( Size: 4 )
+		float m_lastUCmdSimulationRemainderTime; // 0x26ac ( Size: 4 )
+		char _unk46[12]; // 0x26b0 ( Size: 12 )
+		bool m_bShouldDrawPlayerWhileUsingViewEntity; // 0x26bc ( Size: 1 )
+		char _unk47[259]; // 0x26bd ( Size: 259 )
+		int32_t m_autoSprintForced; // 0x27c0 ( Size: 4 )
+		bool m_fIsSprinting; // 0x27c4 ( Size: 1 )
+		char _unk48[7]; // 0x27c5 ( Size: 7 )
+		float m_sprintStartedTime; // 0x27cc ( Size: 4 )
+		float m_sprintStartedFrac; // 0x27d0 ( Size: 4 )
+		float m_sprintEndedTime; // 0x27d4 ( Size: 4 )
+		float m_sprintEndedFrac; // 0x27d8 ( Size: 4 )
+		float m_stickySprintStartTime; // 0x27dc ( Size: 4 )
+		char _unk49[4]; // 0x27e0 ( Size: 4 )
+		int32_t m_ubEFNointerpParity; // 0x27e4 ( Size: 4 )
+		char _unk50[96]; // 0x27e8 ( Size: 96 )
+		char m_title[32]; // 0x2848 ( Size: 32 )
+		char _unk51[252]; // 0x2868 ( Size: 252 )
+		bool m_useCredit; // 0x2964 ( Size: 1 )
+		char _unk52[51]; // 0x2965 ( Size: 51 )
+		float m_smartAmmoPreviousHighestLockOnMeFractionValue; // 0x2998 ( Size: 4 )
+		char _unk53[1292]; // 0x299c ( Size: 1292 )
+		int32_t m_pilotClassIndex; // 0x2ea8 ( Size: 4 )
+		char _unk54[1456]; // 0x2eac ( Size: 1456 )
+		int32_t m_playerScriptNetDataGlobal; // 0x345c ( Size: 4 )
+		char _unk55[5352]; // 0x3460 ( Size: 5352 )
+		int32_t m_selectedOffhand; // 0x4948 ( Size: 4 )
+		char _unk56[1030980]; // 0x494c ( Size: 1030980 )
+		Vector3 m_vecAbsOrigin; // 0x100490 ( Size: 12 )
+		char _unk57[7656]; // 0x10049c ( Size: 7656 )
+		Vector3 m_upDir; // 0x102284 ( Size: 12 )
 	};
-	// clang-format on
+#pragma pack(pop)
+
+	static_assert(offsetof(CBasePlayer, m_nPlayerIndex) == 0x58);
```

Comment:
Definitely good thing to have the static asserts in there as well to ensure the compiler doesn't mess stuff up :D